cql3/query_processor.cc

```diff
@@ -137,6 +137,26 @@ query_processor::query_processor(service::storage_proxy& proxy, database& db, qu
                             _cql_stats.deletes,
                             sm::description("Counts the total number of CQL DELETE requests.")),
 
+                    sm::make_derive(
+                            "system_reads",
+                            _cql_stats.system_reads,
+                            sm::description("Counts the number of CQL read requests executed on SYSTEM keyspace.")),
+
+                    sm::make_derive(
+                            "system_inserts",
+                            _cql_stats.system_inserts,
+                            sm::description("Counts the number of CQL INSERT requests executed on SYSTEM keyspace.")),
+
+                    sm::make_derive(
+                            "system_updates",
+                            _cql_stats.system_updates,
+                            sm::description("Counts the number of CQL UPDATE requests executed on SYSTEM keyspace.")),
+
+                    sm::make_derive(
+                            "system_deletes",
+                            _cql_stats.system_deletes,
+                            sm::description("Counts the number of CQL DELETE requests executed on SYSTEM keyspace.")),
+
                     sm::make_derive(
```

Comment:
Maybe these should be done as a separate label (so we have reads{who="user"} and reads{who="system"}).

Pro:
 - can see all reads by summing across the who label

Con:
 - lots more stats flooding prometheus